src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```diff
@@ -70,6 +83,33 @@ export class RequestFeedbackButtonComponent implements OnInit {
         }
     }
 
+    private subscribeToResultUpdates() {
+        if (!this.participation?.id) {
+            return;
+        }
+
+        // Subscribe to result updates for this participation
+        this.athenaResultUpdateListener = this.participationWebsocketService
+            .subscribeForLatestResultOfParticipation(this.participation.id, true)
+            .pipe(
+                skip(1), // Skip initial value
+                filter((result): result is Result => !!result),
+                filter((result) => result.assessmentType === AssessmentType.AUTOMATIC_ATHENA),
+            )
+            .subscribe(this.handleAthenaAssessment.bind(this));
+    }
+
+    private handleAthenaAssessment(result: Result) {
+        if (result.completionDate) {
+            if (result.successful) {
+                this.currentFeedbackRequestCount += 1;
+                this.alertService.success('artemisApp.exercise.athenaFeedbackSuccessful');
```

Comment:
Is the alert necessary here? I think it is already handled by the corresponding exercise type, right?